src/main/java/electroblob/wizardry/entity/living/EntityIceGiant.java

```diff
@@ -168,8 +168,8 @@ public void onLivingUpdate(){
 	public void onSuccessfulAttack(EntityLivingBase target){
 
 		target.motionY += 0.2;
-		target.motionX += this.getLookVec().xCoord * 0.2;
-		target.motionZ += this.getLookVec().xCoord * 0.2;
+		target.motionX += this.getLookVec().x * 0.2;
+		target.motionZ += this.getLookVec().x * 0.2;
```

Comment:
Probably should use the `z` coordinate here.

Reply:
@gegy1000 : yes, i saw it too, i'll wait electroblob to see what was the correct value :+1:

Reply:
Yes, that should be the z coordinate. Pretty easy to miss since most of the velocity is in the y direction.